test/end-to-end/test/test_editRecipe.js

```diff
@@ -325,8 +325,7 @@ describe('Edit Recipe Page', () => {
         const toastNotifPage = new ToastNotifPage(pageConfig.recipe.simple.name);
```

Comment:
This is unrelated, move it out of here.

Reply:
This case will fail sometimes, and there'll not be coverage result. So I put this improvement here.

Reply:
If this test fails then we don't get "some coverage" but it is not related to the problem of submitting the information to codecov. If you want to keep this change in this PR then at least make 2 separate commits for the two files.